sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuOrcFileFormat.scala

```diff
@@ -44,7 +44,13 @@ object GpuOrcFileFormat extends Logging {
 
   def tagGpuSupport(meta: RapidsMeta[_, _, _],
                     spark: SparkSession,
-                    options: Map[String, String]): Option[GpuOrcFileFormat] = {
+                    options: Map[String, String],
+                    schema: StructType): Option[GpuOrcFileFormat] = {
+
+
+    if(!schema.forall(field => GpuOverrides.isSupportedType(field.dataType))) {
+      meta.willNotWorkOnGpu("Not all datatypes are supported")
```

Comment:
in the very least print the data types here or tell us which one isn't supported